test/helpers/snapshotResolver.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+const webpackPackageJson = require('webpack/package.json');
+
+// eslint-disable-next-line global-require
+const [webpackVersion] = webpackPackageJson.version;
+const snapshotExtension = `.snap.webpack${webpackVersion}`;
```

Comment:
Let's use `.webpack4.snap`, some IDE highlight snap based `snap` extension

Reply:
It does not work on CI, cause jest tries to absolete webpack 5 snapshot (jest finds these by snap-extension) when runs webpack 4 test and throws an error